boot/bootutil/include/bootutil/fault_injection_hardening.h

```diff
@@ -93,9 +93,13 @@ extern "C" {
 #ifndef MCUBOOT_FIH_PROFILE_OFF
 #define FIH_POSITIVE_VALUE 0x1AAAAAAA
 #define FIH_NEGATIVE_VALUE 0x15555555
+#define FIH_CONST1 0x1FCDEA88
+#define FIH_CONST2 0x19C1F6E1
```

Comment:
Is this random or is there some meaning to these constants?

Reply:
Yes, there are random values